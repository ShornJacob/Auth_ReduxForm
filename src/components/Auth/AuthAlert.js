import React from 'react'
import Alert from 'react-bootstrap/Alert'


export default function AuthAlert({variant,message,redirectLink}) {
    return (
            <Alert variant={variant}> {message} {redirectLink ?  <a href={redirectLink} className="alert-link">here.</a> : ''}</Alert>
    )
}
