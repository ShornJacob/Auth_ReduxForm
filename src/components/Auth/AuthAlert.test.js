import React from 'react';
import AuthAlert from './AuthAlert';
import renderer from 'react-test-renderer';


it('renders correctly', () => {

 const code = "ACode"
 const message = "AMessage"


  const tree = renderer
    .create(<AuthAlert code={code} message={message}></AuthAlert>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});



it('renders correctly', () => {

    const code = "ACode"
    const message = "AMessage"
    const redirectLink = "/alink"
   
   
     const tree = renderer
       .create(<AuthAlert code={code} message={message} redirectLink={redirectLink}></AuthAlert>)
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
   
