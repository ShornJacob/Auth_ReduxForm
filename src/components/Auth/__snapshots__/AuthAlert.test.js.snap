// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="fade alert show"
  role="alert"
>
   
  AMessage
   
  
</div>
`;

exports[`renders correctly 2`] = `
<div
  className="fade alert show"
  role="alert"
>
   
  AMessage
   
  <a
    className="alert-link"
    href="/alink"
  >
    here.
  </a>
</div>
`;
